app/Http/Controllers/RepositoryController.php

```
@@ -30,6 +30,10 @@ public function index(): Response
      */
     public function store(StoreRepositoryRequest $request): RedirectResponse
     {
+        if (!$request->user()) {
```

why? not just handle on authorize?

---

Php stan check

---

This or ignore error, a good practice is to avoid check on authorize since we never go on request check something

---

could we do it in other way i think it's not good to have smt like this.